core/cmd/local_client.go

```diff
@@ -735,13 +759,14 @@ func insertFixtures(config config.GeneralConfig, pathToFixtures string) (err err
 // DeleteUser is run locally to remove the User row from the node's database.
 func (cli *Client) DeleteUser(c *clipkg.Context) (err error) {
```

Comment:
I think this is very old and not used, we can probably remove this command entirely to be honest. There is a remote command that does the same thing that supercedes it.

Reply:
It seems in use `core/cmd/app.go:729`

Reply:
Yeah I think we can remove that command,

Reply:
I found this command is used by several tests, looks like this command removal is not as trivial as just removing its code, etc. need to revisit the affected tests. Therefore created that dedicated ticket.